README.md

```diff
@@ -1,3 +1,12 @@
+[comment]: # (#################################################################)
+[comment]: # (Copyright 2017-2120, Lawrence Livermore National Security, LLC)
```

Comment:
2120 -> 2021

Reply:
Thanks. I can't type.  I fixed this and a few others.